sormas-e2e-tests/src/test/java/org/sormas/e2etests/steps/web/application/cases/CaseDetailedTableViewSteps.java

```diff
@@ -141,6 +143,34 @@ public CaseDetailedTableViewSteps(
               "Reporting user is not correct");
           softly.assertAll();
         });
+
+    When(
+        "I back to Case Directory using case list button",
+        () -> webDriverHelpers.clickOnWebElementBySelector(BACK_TO_CASES_LIST_BUTTON));
+
+    When(
+        "I check if Case date format is correct",
+        () -> {
+          List<Map<String, String>> tableRowsData = getTableRowsData();
+          Map<String, String> detailedCaseDTableRow = tableRowsData.get(0);
+          softly.assertTrue(
+              checkDateFormatDE(detailedCaseDTableRow, "NACHVERFOLGUNG BIS"),
+              "Date format is invalid");
+          softly.assertTrue(
+              checkDateFormatDE(detailedCaseDTableRow, "MELDEDATUM"), "Date format is invalid");
+          softly.assertAll();
+        });
+  }
+
+  boolean checkDateFormatDE(Map<String, String> map, String row) {
+    DateFormat dateFormat = new SimpleDateFormat(DATE_FORMAT_DE);
+    dateFormat.setLenient(false);
+    try {
+      dateFormat.parse(map.get(row));
```

Comment:
this method can perform also a dummy parse, if the data is empty, or if the parsing passed but is not as expected, then will return true as well.
Add also an assert here, and a logger to print out what was received and what was parsed, so we can also debug if will fail in jenkins.

Reply:
If parser will have different format than defined in DATE_FORMAT_DE will return false

Reply:
ok, but in this case, we need to assign the parsed value into a variable and check it :D

Reply:
Razvan sorry, i don't get why i need to check something twice? parse fucntion is in try catch, if data format received from website is different than i expect then parser will have ParseException - checked it multiple times. You suggest me to create new variable from parser and check if is correct, it have no sense - if it will be incorrect format, parser will throw exception and that checking will never be called ..

Reply:
yes, you're right, my bad, Just checked on my local.
Then we'll proceed as it is, maybe the logger can be added so we can debug it in jenkins if fails :D

Reply:
ok :) Logger is good idea, i added it